### PR TITLE
Static page

### PR DESCRIPTION
**Pull request by ritamma**

### CODE REVIEW

File: app/controllers/static_pages_controller.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class StaticPagesController < ApplicationController
+  def home
+  end
+  def about
```
Comment (fooooxmr) [inline app/controllers/static_pages_controller.rb:4]: Add enter here.

**Review by fooooxmr — APPROVED** [review]

File: app/views/static_pages/about.html.slim [hunk 2]
```diff
@@ -0,0 +1,2 @@
+h1 About
+p Information about our project
```
Comment (Terapool) [inline app/views/static_pages/about.html.slim:2]: Please, insert here an emty line at the end of file

Comment (ritamma) [reply app/views/static_pages/about.html.slim]: Update, please review

File: app/controllers/static_pages_controller.rb (hunk 1 above)
Comment (ritamma) [reply app/controllers/static_pages_controller.rb]: Update

**Review by fooooxmr — APPROVED** [review]

**Review by Terapool — APPROVED** [review]

**Pull request merged**